packages/app-content-pages/src/screens/About/components/OurMission.js

```diff
@@ -0,0 +1,59 @@
+import { Anchor, Heading, Paragraph } from 'grommet'
+import { Trans, useTranslation } from 'next-i18next'
+import Link from 'next/link'
+
+function LinkInParagraph({ children, href }) {
+  return (
+    <Anchor as={Link} href={href}>
+      {children}
+    </Anchor>
+  )
+}
+
+export default function OurMission() {
+  const { t } = useTranslation()
+  return (
+    <>
+      <Paragraph margin={{ vertical: '20px' }}>
+        <Trans
+          i18nKey='AboutPage.ourMission.paragraphs.one'
```

Comment:
This type of translation component is used from `i18next` to support translated link labels in the middle of a paragraph. In the dictionary file, a few translation strings contain `<0>label<0>`. The text in the middle of the `<0>` corresponds to the `<Link>` passed to `components` prop in `Trans`.